src/plugin.js

```diff
@@ -319,12 +334,12 @@ const eme = function(options = {}) {
       setupSessions(player);
 
       if (window.MediaKeys) {
-        handleEncryptedEvent(player, mockEncryptedEvent, mergedEmeOptions, player.eme.sessions, player.tech_)
+        handleEncryptedEvent(player, mockEncryptedEvent, mergedEmeOptions, player.eme.sessions, player.tech_, emeError)
           .then(() => callback())
           .catch((error) => {
             callback(error);
             if (!suppressErrorIfPossible) {
-              emeError(error);
+              emeError(error, videojs.Error.EMEEncryptedError);
```

Comment:
as discussed offline:
we should probably analyze possible rejections here to narrow it to specific error or remove it